examples/Training/matlab/ROIs.m

```diff
@@ -173,6 +173,124 @@
     roi = session.getUpdateService().saveAndReturnObject(roi);
     fprintf(1, 'Created ROI %g\n', roi.getId().getValue());
 
+    % Create and apply transforms to shapes and add them to ROI
+    
+    % Apply shearing alone to a rectangle (Shear factor set to 10 in the x
+    % and y dimensions in this example)
+    % create rectangle (shape object)
+    rectangle = createRectangle(0, 0, 10, 20);
+    rectangle = setShapeCoordinates(rectangle, 0, 0, 0);
```

Comment:
@will-moore : setShapeCoordinates sets the Z,C and T for the shapes. And it is setting them to 0,0,0 in this case!